Sources/PaymentsCore/CoreConfig.swift

```diff
@@ -3,6 +3,16 @@ import Foundation
 /// The configuration object containing information required by every payment method.
 /// It is used to initialize all Client objects.
 public struct CoreConfig {
+
     public let clientID: String
     public let environment: Environment
+
+    /// Return URL required for PayPal integration
+    public let returnUrl: String?
```

Comment:
Sure - we can put a pin in this. We should definitely investigate trying to hide this from the merchant - we would need to work with the team behind the dashboard (auth/identity?) to figure out if we can maybe use 1 URL fo every merchant instead.

Reply:
I also just don't really understand how / why this URL is used on the backend side. Maybe we can do a pow-wow on it soon and figure out who to talk to next.

Reply:
Yeah this is used by identity to verify that the url provided matches what exists in the dev portal to avoid malicious redirects. This doesn't really apply to an iOS environment - and we could absolutely use a single `returnURL` for every merchant (we would need to set this value from the BE when the user is integrating with / opting into the NXO SDK). 

On Android, this would be more difficult, since the chrome custom tab they are using actually redirects to the specified returnURL (rather than just dismissing, which is what the auth sdk does natively on iOS) - so the return url has to be their app identifier (@tdchow could articulate this better)

Reply:
I'm all for obscuring this from the merchant however we can. It's been the source of a lot of integration issues for merchants.